src/containers/ThreadsDropdown.js

```diff
@@ -109,16 +109,23 @@ export default class ThreadsDropdown extends React.Component {
 
     if (this.state.open || thisThreadOpen) return dispatch(appendComment(postId, message))
 
-    if (!threads.length) { // or you don't have the thread in question?
```

Comment:
As of now, the comment/question here should never be the case, because if you're connected to the socket it will have refreshed the thread list. If you're not connected to the socket this event can't fire